### ISSUE TITLE
Multiple Controls On Same Pop-over

### ISSUE THREAD

**Issue by artforlife** [vitalets/angular-xeditable#535]
Is this possible to implement? That is, it is probably implemented and I am wondering about how to do it.

For instance, I would something along the lines of this, except with two select fields and labels for each one. 

https://vitalets.github.io/angular-xeditable/#/select-local

**Comment by ckosloski** [comment]
No, currently you can only do the pop-over for one xeditable element.

**Comment by artforlife** [comment]
What about the Time format? There are clearly several controllers on a single popover. Is this built-into the x-editable? 

What I am after is something that will edit one field, but at least allow for labels to be included on the pop-over. It would also be cool if one could attach several fields to be edited to one pop-over (that is triggered from a single link in DOM).

**Comment by ckosloski** [comment]
All single editable elements will work in a pop-over.  There currently is nothing to create a form in a pop-over or string multiple elements together in a pop-over.

Pull requests are welcome :)

**Comment by artforlife** [comment]
Let me see what I can do. I have not looked at the code yet. But I do need this for myself, so ...  :)

**Comment by ckosloski** [comment]
@artforlife did you ever take a look at the code for a PR?

**Comment by ckosloski** [comment]
Take a look at [this](http://jsfiddle.net/ckosloski/dkbhc9s5/13/) and see if it works for you.

**Comment by ckosloski** [comment]
@artforlife can this be closed?

**Comment by santoshgiridhar** [comment]
@ckosloski  - How do we validate the fields inside popup? (without closing the popup - [example](http://jsfiddle.net/santoshgiridhar/rpya0znm/))

**Comment by ckosloski** [comment]
You need to return false from your `onbeforesave` method
http://vitalets.github.io/angular-xeditable/#onbeforesave
